OntologyMapper/src/Mappings/Mapped/Json/v0/BrickRec/mapped_json_v0_dtdlv3_Brick_1_3-REC_4_0.json

```diff
@@ -0,0 +1,5494 @@
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Wing;1",
+      "OutputDtmi": "dtmi:org:w3id:rec:BuildingElement;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:WirelessAccessPoint;1",
+      // Invalid Output DTMI - Review Needed
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Wireless_Access_Point;1",
+      "IsIgnored": true
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Workshop;1",
+      "OutputDtmi": "dtmi:org:w3id:rec:Workshop;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Zone;1",
+      "OutputDtmi": "dtmi:org:w3id:rec:Zone;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:ZoneAirCoolingTemperatureSetpoint;1",
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Air_Cooling_Temperature_Setpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:ZoneAirDewpointSensor;1",
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Air_Dewpoint_Sensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:ZoneAirHeatingTemperatureSetpoint;1",
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Air_Heating_Temperature_Setpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:ZoneAirHumiditySensor;1",
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Air_Humidity_Sensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:ZoneAirHumiditySetpoint;1",
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Air_Humidity_Setpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:ZoneAirTemperatureSensor;1",
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Air_Temperature_Sensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:ZoneAirTemperatureSetpoint;1",
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Air_Temperature_Setpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:ZoneConditioningModeStatus;1",
+      // Mapped to Base - Review needed
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Status;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:ZoneStandbyLoadShedCommand;1",
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Standby_Load_Shed_Command;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:ZoneUnoccupiedLoadShedCommand;1",
+      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Unoccupied_Load_Shed_Command;1"
+    }
+  ],
+  "RelationshipRemaps": [
+    {
+      "InputRelationship": "floors",
+      "OutputRelationship": "hasPart"
+    },
+    {
+      "InputRelationship": "hasPart",
+      "OutputRelationship": "isLocationOf"
+    }
+  ],
+  "PropertyProjections": [
+    {
+      "OutputDtmiFilter": "*",
+      "OutputPropertyName": "externalIds",
+      "InputPropertyName": "mappingKey",
+      "IsOutputPropertyCollection": true
+    }
+  ],
+  "FillProperties": [
+    {
+      "OutputDtmiFilter": "*",
+      "OutputPropertyName": "name",
+      "InputPropertyNames": "name description"
```

Comment:
How can more than one Input fill a single Output? Should this be an array?

Reply:
Converted to an array